### PR TITLE
Remove image now not coming when user changes fullname. Fixes #17320

### LINKED ISSUE #17320: "Remove image" button appears in the avatar section of the personal page just by changing the full name of the user
### Steps to reproduce
1. Go to the personal page
2. Delete any avatar you might have
3. Change the full name of the user

### Expected behaviour
The avatar section should remain unchanged

### Actual behaviour
A "remove image" button appears in the avatar section even though the avatar is the default one

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4.7

**Database:** MySQL

**PHP version:** 5.5.9

**ownCloud version:** OC 8.1RC2

**Updated from an older ownCloud or fresh install:** fresh install

### PR DESCRIPTION
**Pull request by patelfenil**
Fixes #17320

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
requires rebase

**Comment by patelfenil** [comment]
@DeepDiver1975  rebase done

**Comment by patelfenil** [comment]
@Kondou-ger, @nickvergessen and @icewind1991 Please review

**Comment by nickvergessen** [comment]
@patelfenil can you please take care of the comment from owncloud-bot?

File: settings/js/personal.js [hunk 1]
```diff
@@ -107,8 +107,16 @@ function updateAvatar (hidedefault) {
...
+	$.get(OC.generateUrl(
+		'/avatar/{user}/{size}',
+		{user: OC.currentUser, size: 1}
+	), function (result) {
+		if (typeof(result) === 'string') {
+			// Show the delete button when the avatar is custom
+			$('#removeavatar').removeClass('hidden').addClass('inlineblock');
+		}
+	});
+	//$('#removeavatar').removeClass('hidden').addClass('inlineblock');
```
Comment (nickvergessen) [inline settings/js/personal.js]: just delete this

**Comment by patelfenil** [comment]
@nickvergessen Taken care of the comment. Please review. @karlitschek can you confirm here?

**Comment by rullzer** [comment]
Ran js unit test locally and all still works.
Fixes the issue :+1:

**Comment by PVince81** [comment]
@karlitschek did you receive the CLA from @patelfenil ?

**Comment by karlitschek** [comment]
@PVince81 @patelfenil Yes. Confirmed and he is a member of the core developer group!

**Comment by PVince81** [comment]
:+1: looks good.

I ran the JS unit tests locally, all still pass. AFAIK there was no unit tests yet for this file.

**Pull request merged**